independent-publisher-connectors/Shopify Admin Store/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2373 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Store",
+    "description": "Manage a shopify store's customers, products, collections and customers using the Shopify Admin API.",
+    "contact": {
+      "name": "Hitachi Solutions",
+      "email":"jlindstrom@hitachisolutions.com",
+      "url": "https://global.hitachi-solutions.com/"
+    }
+  },
+  "host": "storename.myshopify.com",
+  "basePath": "/admin/api/2021-04",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/orders.json": {
+      "post": {
+        "description": "Creates an order",
+        "summary": "Create an order",
+        "tags": [
+          "Order"
+        ],
+        "operationId": "CreateOrder",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Body",
+            "in": "body",
+            "required": true,
+            "description": "Order details",
+            "schema": {
+              "$ref": "#/definitions/CreateapendingorderwithanexistingcustomerRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@thatapiguy Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**